exp/operator/config/certmanager/certificate.yaml

```diff
@@ -1,6 +1,6 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for 
+# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for
 # breaking changes
 apiVersion: cert-manager.io/v1alpha2
```

Comment:
We should switch these to `cert-manager.io/v1` - same as they are on the master branch. `v1alpha2` seems quite outdated to me.

Reply:
right, the same story as with logger, can be done in a follow-up PR, I'll remove this blank space fix too. This is also tracked here https://github.com/kubernetes-sigs/cluster-api/issues/4246